config/tectonic-ingress/config.go

```diff
@@ -0,0 +1,20 @@
+package tectonicingress
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// Kind is the TypeMeta.Kind for the OperatorConfig.
+	Kind = "TectonicIngressOperatorConfig"
+	// APIVersion is the TypeMeta.APIVersion for the OperatorConfig.
+	APIVersion = "v1"
+)
+
+// OperatorConfig defines the configuration needed by the Tectonic Network Operator.
+type OperatorConfig struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// The installer platform, e.g. "bare-metal"
+	InstallerPlatform string `json:"installerPlatform"`
```

Comment:
Is this different from `CloudProviderConfig` used elsewhere? c.f.

https://github.com/coreos/tectonic-config/blob/master/config/kube-core/config.go#L35
https://github.com/coreos/tectonic-installer/blob/master/modules/bootkube/resources/kco-config.yaml#L10
https://github.com/coreos/tectonic-installer/blob/master/modules/bootkube/assets.tf#L54

Reply:
Yeah. cloud_provider is 'metal' for metal, gcp, vmware, and openstack.

Reply:
Ok, to be more concrete: how will this be filled in in the installer (last 2 links)? Can't we just use the CloudProviderProfile and then add the relevant switch statement in the operator?

We don't want to add a *new* variable / conditional logic in the installer--we just want to reuse parameters that are already there.

Reply:
Ah InstallerPlatform alreay exists, never mind.